js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -3305,3 +3305,5 @@ filterByMembershipType=Filter by Membership Type
 organizationsMembersListError=Could not fetch organization members\: {{error}}
 MANAGED=Managed
 UNMANAGED=Unmanaged
+removeInvalidUsers=Remove invalid users
+removeInvalidUsersHelp=Remove invalid or deleted LDAP users from the KC database. Should be true otherwise invalid and deleted users stay in the KC database. Set to false if you have problems with your LDAP setup and valid users get deleted.
```

```suggestion
removeInvalidUsersHelp=Remove invalid or deleted users from the KC database, which are not found in user storage. If this is true, then users, which were deleted from 3rd party user storage (EG. LDAP) will be deleted as well from the KC database whenever an attempt to lookup particular user is performed from KC. If false, then users previously imported from 3rd party user storage will be still kept in the KC database even if KC found that user is not anymore in the 3rd party user storage (for example user was deleted directly from LDAP). Set to false if you have problems with setup of your user storage (EG. setup of your LDAP server) and valid users are accidentally deleted from KC database.
```

I prefer to have slightly more descriptive tooltip, so it is more clear what this switch is doing for someone, who is seeing this for the first time. Also the switch is potentially not specific to LDAP, so would prefer some more generic text, so people can use it in their providers. How about the text above?